CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+7.0.0 / 2020-01-22
+===================
+- Added logic for the is_element_visible function which reduces implicit wait when no wait parameter is passed to the
+function
+- Required change in any test packs
```

Comment:
I think this description should be simplified to state that you have changed the is_element_visible function to avoid using the implicit wait when waiting is not required, to speed up test runs.